data/translations/kdiskmark_es_MX.ts

```diff
@@ -0,0 +1,462 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1">
+<context>
+    <name>About</name>
+    <message>
+        <location filename="../../src/about.ui" line="17"/>
+        <source>About KDiskMark</source>
+        <translation>Acerca de KDiskMark</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="62"/>
+        <source>Version:</source>
+        <translation>Versión:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="88"/>
+        <source>License:</source>
+        <translation>Licencia:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="114"/>
+        <source>Country:</source>
+        <translation>País:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="127"/>
+        <source>Russia</source>
+        <translation>Rusia</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="140"/>
+        <source>Author:</source>
+        <translation>Autor:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="172"/>
+        <source>E-mail:</source>
+        <translation>Correo Electrónico:</translation>
+    </message>
+    <message>
+        <location filename="../../src/about.ui" line="204"/>
+        <source>Flexible I/O Tester:</source>
+        <translation>Probador de E/S Flexible</translation>
```

Comment:
Flexible I/O Tester is a tool, I don't think it needs to be translated, also it's just a too long message.